.github/workflows/release.yaml

```diff
@@ -40,7 +40,9 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-      - run: uv run python scripts/generate_wheels.py
+      - run: |
+          uv sync --frozen
```

Comment:
BTW since `uv run` automatically syncs, maybe this isn't needed actually

Reply:
yeah, I see..  I’m not sure if --frozen is actually useful in this case.
Anyway, for now, I will merge this PR now